### PR TITLE
Update UT3GoliathProjectile.uc

### PR DESCRIPTION
**Pull request by HellDragon-HK**

### CODE REVIEW

File: Classes/UT3DmgType_GoliathTankShell.uc [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2014 GreatEmerald
```
Comment (GreatEmerald) [inline Classes/UT3DmgType_GoliathTankShell.uc]: Fix copyright

File: Classes/UT3DmgType_GoliathTankShell.uc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
+ * IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software is not supported by Atari, S.A., Epic Games, Inc. or any
+ * of such parties' affiliates and subsidiaries.
+ */
+
+class UT3DmgType_GoliathTankShell extends VehicleDamageType;
```
Comment (GreatEmerald) [inline Classes/UT3DmgType_GoliathTankShell.uc]: Extend the Goliath damage type in UT2004; you will then be able to remove a lot of the redundant properties

File: Classes/UT3DmgType_GoliathTankShell.uc [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+defaultproperties
+{
+    DeathString="%k's tank shell blew %o into flaming bits."
+    MaleSuicide="%o fired his rocket prematurely."
+    FemaleSuicide="%o fired her rocket prematurely."
```
Comment (GreatEmerald) [inline Classes/UT3DmgType_GoliathTankShell.uc]: Double-check these

File: Classes/UT3DmgType_GoliathTankShell.uc [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    
+    bDetonatesGoop=true
+    KDamageImpulse=8000
+    VehicleMomentumScaling=1.5
+    bThrowRagdoll=true
+    GibPerterbation=0.15
+    bFlaming=true
+    bDelayedDamage=true
```
Comment (GreatEmerald) [inline Classes/UT3DmgType_GoliathTankShell.uc]: A lot of these properties are probably already in the parent class if you extend the Goliath damage type

File: Classes/UT3GoliathProjectile.uc [hunk 5]
```diff
@@ -2,12 +2,14 @@
 // UT3GoliathProjectile.uc
 // Yeap, it's here.
 // GreatEmerald, 2008
+// HellDragon (Just added the DmgType and changed Radius to UT3 value is all I've done here)
```
Comment (GreatEmerald) [inline Classes/UT3GoliathProjectile.uc]: Use the `Copyright ©` format and don't put comments

File: Classes/UT3GoliathProjectile.uc [hunk 6]
```diff
@@ -1,13 +1,15 @@
...
-// UT3GoliathProjectile.uc
-// Yeap, it's here.
-// GreatEmerald, 2008
-//-----------------------------------------------------------
+/*-----------------------------------------------------------
+* UT3GoliathProjectile.uc
+* Yeap, it's here.
+* Copyright © 2014 GreatEmerald
+* Copyright © 2017 HellDragon
+*/-----------------------------------------------------------
```
Comment (GreatEmerald) [inline Classes/UT3GoliathProjectile.uc]: This will not compile, there cannot be anything after the `*/`.

**Comment by GreatEmerald** [comment]
I'm pretty sure: https://github.com/HellDragon-HK/UT3Vehicles/blob/06217bde5f1ec9e501c62c2e010c3351b9d7cc17/Classes/UT3GoliathProjectile.uc The `-`s should be grey, not red!

**Comment by HellDragon-HK** [comment]
Oh ok I see what you mean sorry

**Comment by GreatEmerald** [comment]
OK, thanks.

**Pull request merged**